include/cellular_api.h

```diff
@@ -40,12 +41,14 @@
  *
  * @param[in,out] pCellularHandle the handle pointer to store the cellular handle.
  * @param[in] pCommInterface Comm interface for communicating with the module.
+ * @param[in] pTokenTable Token tables for pkthandler and pktio.
  *
  * @return CELLULAR_SUCCESS if the operation is successful, otherwise an error
  * code indicating the cause of the error.
  */
 CellularError_t Cellular_Init( CellularHandle_t * pCellularHandle,
-                               const CellularCommInterface_t * pCommInterface );
+                               const CellularCommInterface_t * pCommInterfac,
+                               const CellularTokenTable_t * pTokenTable );
```

Comment:
Cellular Library APIs user don't need to know cellular module token table.
This information should stay in cellular common APIs.
I suggest we don't change Cellular_Init prototype but we change Cellular_CommonInit prototype only.

Cellular module implements their Cellular_Init with the following code.
```

/* Token is module defined. It is better we put it in cellular common library only. */
static const CellularTokenTable_t CellularTokenTable = ...;

CellularError_t Cellular_Init( CellularHandle_t * pCellularHandle,
                               const CellularCommInterface_t * pCommInterface )
{
    return Cellular_CommonInit( pCellularHandle, pCommInterface, &CellularTokenTable );

}
```